test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -1025,6 +1027,40 @@ def test_qconfig_dict_object_type_module(self):
         self.assertTrue(isinstance(mq[1], nn.Hardswish))
         self.assertTrue(isinstance(mq[2], nnq.Conv2d))
 
+    def test_qconfig_dict_object_type_function(self):
+        """
+        Verifies that the 'object_type' option of qconfig_dict works
+        on function types.
+        """
+        class M(nn.Module):
+            def forward(self, x):
+                x = x + x
+                x = x * x
+                return x
+
+        m = M()
+        # TODO(future PR): also implement global qconfig being None
+        # and individual functions having qconfigs
+        qconfig_dict = {
+            '': torch.quantization.default_qconfig,
+            'object_type': [
+                (torch.add, None),
+            ],
+        }
+        example_args = (torch.randn(1, 1, 1, 1),)
+        mp = _quantize_dbr.prepare(m, qconfig_dict, example_args)
+        mp(*example_args)
+        mq = _quantize_dbr.convert(mp)
+        mq(*example_args)
+        rewritten = mq.rewrite_for_scripting()
```

Comment:
just curious, is there a way to check what ops are run in dbr quantized models? or do we plan to build some utilities to collect that information?

Reply:
Currently the developer can turn on logging (by hardcoding a flag in `auto_trace.py`) to inspect it in real time.  I think the cleanest way to do it in tests is to encode the transforms into the graph and check the graph, this can be done with `torch.jit.trace` or `torch.jit.script`.  The current code uses the FX bridge for `torch.jit.script`.

I'm flexible if we want to do something else here, lmk.